k8_cortx_cloud/logs-cortx-cloud.sh

```diff
@@ -85,14 +85,16 @@ function savePodDetail()
 function getInnerLogs()
 {
   local pod="$1"
+  local container_arg=()
+  [[ -n $2 ]] && container_arg+=(--container "$2")
```

Comment:
This is probably not really needed, there shouldn't be any pod w/o a container, but in case something strange happened this would just go with the previous behavior of not specifying a container. I can remove it if it seems pointless.

Reply:
I'm good with this being added. I'd rather add it to explicitly catch an issue that would otherwise be thrown to the output and missed by the user.